ethereum_history_api/circuits/lib/src/verifiers/receipt.nr

```diff
@@ -1,12 +1,14 @@
 use dep::proof::{
-    const::{HASH_LENGTH, MAX_RECEIPT_RLP_LENGTH as MAX_ENCODED_RECEIPT_LENGTH}, trie_proof::TrieProof,
-    rlp::{RLP_List, decode1, decode0, STRING, LIST}, utils::{byte_value, memcpy}
+    const::HASH_LENGTH, trie_proof::TrieProof, rlp::{RLP_List, decode1, decode0, STRING, LIST},
+    utils::{byte_value, memcpy}
 };
-use crate::receipt::{TxReceiptProof, TxReceiptPartial, MAX_RECEIPT_TREE_DEPTH, TxType, BLOOM_FILTER_LEN};
+use crate::receipt::{TxReceiptPartial, MAX_RECEIPT_TREE_DEPTH, TxType, BLOOM_FILTER_LEN};
+use dep::proof::receipt_proof::ReceiptProof;
 
 global RECEIPT_FIELDS_COUNT = 4;
 
 global BYZANTIUM_BLOCK_NUM: u64 = 4_370_000;
+global MAX_ENCODED_RECEIPT_LENGTH: u64 = 525;
```

Comment:
Just moving here constant previously imported from library where it no longer exists, but I don't want to go into refactoring thin in this PR. Ultimately this should be a generic parameter extracted to library client. Created [task](https://www.notion.so/vlayer/Receipt-Make-MAX_ENCODED_RECEIPT_LENGTH-generic-parameter-008044df8ee749908398988c31c679c8?pvs=4) for that